packages/market/client/utils.ts

```diff
@@ -80,7 +80,7 @@ function getSimilarity(data: SearchObject, word: string) {
   if (shortname.includes(word)) return 2
   return [
     ...data.package.keywords,
-    ...Object.values(data.manifest.description),
+    ...Object.values(data?.manifest?.description ?? {}),
```

Comment:
```suggestion
    ...Object.values(data.manifest?.description ?? {}),
```

`data` cannot be `undefined`, otherwise the above line will throw.